### PR TITLE
Improve packaging + add ROS package

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -111,8 +120,6 @@ IF(OCTOMAP_FOUND)
     -DOCTOMAP_PATCH_VERSION=${OCTOMAP_PATCH_VERSION})
 ENDIF(OCTOMAP_FOUND)
 
-target_include_directories(${LIBRARY_NAME} PUBLIC
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
```
Comment (jmirabel) [inline src/CMakeLists.txt]: You should not remove this. It adds to the exported target the include directory so that `target_link_libraries` can use it.

Comment (jmirabel) [reply src/CMakeLists.txt]: Ah, I see now that it was only moved.

Comment (jcarpent) [reply src/CMakeLists.txt]: Yes, I wanted to gather lines together ;).

**Comment by jcarpent** [comment]
@jmirabel I think after Travis is happy, you can proceeed to the merge if you think it is fine.

**Pull request merged**